includes/REST_API/Stories_Controller.php

```diff
@@ -270,128 +289,137 @@ public function get_items( $request ) {
 			return $response;
 		}
 
-		// Retrieve the list of registered collection query parameters.
-		$registered = $this->get_collection_params();
-		$args       = [];
-
-		/*
-		 * This array defines mappings between public API query parameters whose
-		 * values are accepted as-passed, and their internal WP_Query parameter
-		 * name equivalents (some are the same). Only values which are also
-		 * present in $registered will be set.
-		 */
-		$parameter_mappings = [
-			'author'         => 'author__in',
-			'author_exclude' => 'author__not_in',
-			'exclude'        => 'post__not_in', // phpcs:ignore WordPressVIPMinimum.Performance.WPQueryParams.PostNotIn, WordPressVIPMinimum.Performance.WPQueryParams.PostNotIn_exclude
-			'include'        => 'post__in',
-			'menu_order'     => 'menu_order',
-			'offset'         => 'offset',
-			'order'          => 'order',
-			'orderby'        => 'orderby',
-			'page'           => 'paged',
-			'parent'         => 'post_parent__in',
-			'parent_exclude' => 'post_parent__not_in',
-			'search'         => 's',
-			'slug'           => 'post_name__in',
-			'status'         => 'post_status',
-		];
+		$response = $this->add_response_headers( $response, $request );
+		if ( is_wp_error( $response ) ) {
+			return $response;
+		}
 
-		/*
-		 * For each known parameter which is both registered and present in the request,
-		 * set the parameter's value on the query $args.
-		 */
-		foreach ( $parameter_mappings as $api_param => $wp_param ) {
-			if ( isset( $registered[ $api_param ], $request[ $api_param ] ) ) {
-				$args[ $wp_param ] = $request[ $api_param ];
-			}
+		if ( $request['_web_stories_envelope'] ) {
+			$embed    = isset( $request['_embed'] ) ? rest_parse_embed_param( $request['_embed'] ) : false;
+			$response = rest_get_server()->envelope_response( $response, $embed );
 		}
 
-		// Check for & assign any parameters which require special handling or setting.
-		$args['date_query'] = [];
+		return $response;
+	}
 
-		// Set before into date query. Date query must be specified as an array of an array.
-		if ( isset( $registered['before'], $request['before'] ) ) {
-			$args['date_query'][0]['before'] = $request['before'];
-		}
+	/**
+	 * Prepares the 'tax_query' for a collection of posts.
+	 *
+	 * Remove this method, once WP 5.7 is min required.
+	 *
+	 * @SuppressWarnings(PHPMD.NPathComplexity)
```

Comment:
This method is copied from core. I am not going to refactor it. So this can not be avoided.